pkg/provision/providers/docker/docker.go

```diff
@@ -85,11 +85,11 @@ func (p *provisioner) GetLoadBalancers(networkReq provision.NetworkRequest) (int
 	case "darwin", "windows":
 		return "", "127.0.0.1"
```

Comment:
this function should always return `"", "127.0.0.1"` no matter which runtime.GOOS to make it consistent

Reply:
And also perhaps we should make Docker only listen on 127.0.0.1?

Reply:
I didn't get that. Inside the container it will listen on a container port, outside the container should be only exposed to the localhost.

Reply:
Yes, but currently it forwards to 0.0.0.0 using `50000:50000/tcp` and so on